graphstorm-processing/tests/test_dist_heterogenous_loader.py

```diff
@@ -1033,3 +1033,119 @@ def test_edge_custom_label(spark, dghl_loader: DistHeterogeneousGraphLoader, tmp
     assert train_total_ones == 3
     assert val_total_ones == 3
     assert test_total_ones == 3
+
+
+def test_node_custom_label_multitask(spark, dghl_loader: DistHeterogeneousGraphLoader, tmp_path):
```

Comment:
Are we testing for corrected order somewhere?

Reply:
Where does `tmp_path` get its value?

Reply:
Built-in value for pytest, check here: https://docs.pytest.org/en/stable/how-to/tmp_path.html